src/NonCumulative/Semantics/Properties/PER.agda

```diff
@@ -469,6 +469,48 @@ El-refl {i = i} A≈B a≈b = El-one-sided (𝕌-trans A≈B (𝕌-sym A≈B)) A
                                                  a≈b
                                                  (El-sym A≈B (𝕌-sym A≈B) a≈b))
 
+El-L-𝕌 : ∀ {i j k}
+           (A≈A′ : A ≈ A′ ∈ 𝕌 k) →
+           (i≡j+k : i ≡ j + k) →
+           a ≈ a′ ∈ El _ (L-𝕌 A≈A′ i≡j+k) →
+           a ≈ a′ ∈ Unli (El _ A≈A′)
+El-L-𝕌 {i = i} {j = j} {k = k} kA eq a∈
+  with (L-𝕌 kA eq)
+El-L-𝕌 {i = i} {j = j} {k = k} kA eq record { ua = ua ; ub = ub ; ↘ua = ↘ua ; ↘ub = ↘ub ; ua≈ub = ua≈ub } | L i≡′j+k kA′ _ _
+  rewrite 𝕌-wf-gen k (Li≤′ j k i≡′j+k) = record { ua = ua ; ub = ub ; ↘ua = ↘ua ; ↘ub = ↘ub ; ua≈ub = El-one-sided kA′ kA ua≈ub }
+
+El-Π-𝕌 : ∀ {i j k} →
```

Comment:
this looks exactly the second projection of `Π-bundle` to me. What's the difference? Same for L,

Reply:
I think it's necessary because the `El _ (L-𝕌 A≈A′ i≡j+k)` would not be automatically reduced by Agda since it does not match the definition of the `L` case in `PERDef.El`. We can discuss this in detail next meeting.